README.md

```diff
@@ -4,6 +4,8 @@
 
 The connector uses the [Spark SQL Data Source API](https://spark.apache.org/docs/latest/sql-programming-guide.html#data-sources) to read data from [Google BigQuery](https://cloud.google.com/bigquery/).
 
+It does not support writing data to [Google BigQuery](https://cloud.google.com/bigquery/).
```

Comment:
s/"It"/"The connector", or else merge this into the paragraph above.

Reply:
Done. Also added link to the other BQ connector which supports write.